docs/user-interface/controls/graphicsview.md

```diff
@@ -0,0 +1,96 @@
+---
+title: ".NET MAUI GraphicsView"
+description: "The .NET MAUI GraphicsView is a graphics canvas on which 2D graphics can be drawn using the cross-platform Microsoft.Maui.Graphics library."
+ms.date: 07/28/2021
+---
+
+# .NET MAUI GraphicsView
+
+<!-- Sample link goes here -->
+
+The .NET Multi-platform App UI (.NET MAUI) `GraphicsView` is a graphics canvas on which 2D graphics can be drawn using the `Microsoft.Maui.Graphics` library. For more information about `Microsoft.Maui.Graphics`, see [.NET MAUI Graphics](~/user-interface/graphics/index.md).
+
+`GraphicsView` defines the `Drawable` property, of type `IDrawable`, which specifies the content that will be drawn. This property is backed by a `BindableProperty`, which means it can be the target of data binding, and styled.
+
+## Create a GraphicsView
+
+A `GraphicsView` must define an `IDrawable` object that specifies the content that will be drawn on the control. This can be achieved by creating an object that derives from `IDrawable`, and by implementing its `Draw` method:
+
+```csharp
+using Microsoft.Maui.Graphics;
+
+namespace MyMauiApp
+{
+    public class GraphicsDrawable : IDrawable
+    {
+        public void Draw(ICanvas canvas, RectangleF dirtyRect)
+        {
+            // Drawing code goes here
+        }      
+    }
+}
+```
+
+The `Draw` method has `ICanvas` and `RectangleF` arguments. The `ICanvas` argument is the drawing canvas on which you draw graphical objects. The `RectangleF` argument is a `struct` that contains data about the size and location of the drawing canvas. For more information about drawing on an `ICanvas`, see [.NET MAUI Graphics: Draw objects](~/user-interface/graphics/draw.md).
```

Comment:
There are a few instances of a site relative link `~/path/article` but we're not really supposed to use those unless required (such as if the publishing config imports a repo in here and you're linking to that cross-repo content). You can't click on them and get them working in vscode nor in github directly.

Reply:
To be cleaned up later by a script.